app/src/main/res/menu/nav_menu.xml

```diff
@@ -4,25 +4,28 @@
   ~ License, v. 2.0. If a copy of the MPL was not distributed with this
   ~ file, You can obtain one at http://mozilla.org/MPL/2.0/.
   -->
-<menu xmlns:app="http://schemas.android.com/apk/res-auto"
-    xmlns:android="http://schemas.android.com/apk/res/android">
+<menu xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_height="match_parent"
+    android:layout_width="match_parent">
 
-    <group android:checkableBehavior="none">
+    <group android:checkableBehavior="none"
+           android:layout_height="match_parent"
+           android:layout_width="wrap_content">
         <item
             android:id="@+id/account_setting_menu_item"
             android:icon="@drawable/ic_account"
-            android:title="@string/nav_menu_account" />
+            android:title="@string/nav_menu_account"/>
         <item
             android:id="@+id/faq_menu_item"
-            android:icon="@drawable/ic_help_faq"
+            android:icon="@drawable/ic_icon_faq"
             android:title="@string/nav_menu_faq" />
         <item
-            android:id="@+id/setting_menu_item"
-            android:icon="@android:drawable/ic_menu_preferences"
-            android:title="@string/nav_menu_settings" />
+            android:id="@+id/feedback_menu_item"
+            android:icon="@drawable/ic_icon_feedback"
```

Comment:
naming nit: `ic` stands for `icon`, so here all of your icons are named `icon_icon_feedback`. would rather see `ic_feedback`.

Reply:
good call! I'll rename these, and also the rest of the icons that have the same "ic_icon" in their names :)